src/index.js

```diff
@@ -11,13 +11,11 @@ import store from './ducks/store';
 //Routing
 import { BrowserRouter as Router } from 'react-router-dom'; 
 
-ReactDOM.render(<Provider store={store}>
-    <Router>
-    <App />
-    </Router>
+ReactDOM.render(
+    <Provider store={store}>
```

Comment:
Just prettied this up a bit to have good indentation.  This will be important if we keep wrapping in stuff.